vertical-pod-autoscaler/pkg/recommender/model/vpa_test.go

```diff
@@ -51,6 +51,7 @@ func TestUpdateConditions(t *testing.T) {
 		podsMatched        bool
 		hasRecommendation  bool
 		expectedConditions []vpa_types.VerticalPodAutoscalerCondition
+		expectedAbsent     []vpa_types.VerticalPodAutoscalerConditionType
```

Comment:
I think a real test for this would be to call UpdateConditions twice, otherwise the default behaviour (not touching the condition) will pass this check.
Feel free to keep it for a later PR.